src/index.html

```diff
@@ -22,6 +22,536 @@
+            <a
+              href="./index.html"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="./index.html"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- header ends  -->
+
+    <!-- cards starts -->
+    <main class="cards">
+      <!-- card #1 -->
+      <section
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          class="product-card__product-image"
+          src="./images/imac.png"
+          alt="apple-imac-27"
+        />
+
+        <div class="product-card__product-details">
+          <h3 class="product-card__product-details-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="product-card__product-details-description">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="product-card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+            <div class="stars__star">
+              <div class="stars__star-image"></div>
+            </div>
+          </div>
+
+          <p class="product-card__rating-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+
+          <p class="product-card__price-cost">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="product-card__button-buy"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </section>
+
+      <!-- card #2 -->
+      <section class="product-card">
+        <img
+          class="product-card__product-image"
+          src="./images/imac.jpeg"
```

Comment:
The image source should be consistent across all product cards. It's referencing `imac.jpeg` here while the first card references `imac.png`. Please correct this inconsistency.